tests/tests_pytorch/utilities/test_parsing.py

```diff
@@ -36,7 +36,7 @@
 unpicklable_function = lambda: None
 
 
-def model_cases():
```

Comment:
Are these changes supposed to be in here?

Reply:
I added them to see if https://github.com/Lightning-AI/lightning/pull/15272 fixes them. That PR should land first of course